src/foam/u2/md/OverlayDropdown.js

```diff
@@ -152,8 +152,9 @@ foam.CLASS({
       this.close();
     },
 
-    function onMouseLeave(e) {
-      console.assert(e.target === this.dropdownE_.el(),
+    async function onMouseLeave(e) {
+      var el = await this.dropdownE_.el();
```

Comment:
I think we could use el_() here because if we received a mouseLeave event, then the element must be active in the DOM.